README.md

```diff
@@ -1,7 +1,7 @@
 # sealed-enum (Beta)
 *Enums are dead, long live enums!*
 
-![CI-Status](https://travis-ci.com/livefront/sealed-enum.svg?token=WuckG7RmGXoNTzG4QzVk&branch=master)
+![CI](https://github.com/livefront/sealed-enum/workflows/CI/badge.svg)
```

Comment:
I assume it will look better than this?

<img width="379" alt="Screen Shot 2021-02-08 at 10 46 39 AM" src="https://user-images.githubusercontent.com/5686479/107252432-f51d2300-69fa-11eb-99e3-9a7b1f1bb48b.png">

Reply:
Yep! That's the diff view, after the change the old one is removed completely:

https://github.com/livefront/sealed-enum/blob/4d4401c3e39dcf22d84ca38ee79c5517e63e12e0/README.md

![Screen Shot 2021-02-08 at 12 05 11 PM](https://user-images.githubusercontent.com/4217560/107262238-e720cf80-6a05-11eb-8d4a-cdc67aae7434.png)